external_systems/sources/_sources.py

```diff
@@ -75,13 +75,20 @@ def client_certificate(self) -> Optional[ClientCertificateFilePaths]:
     def _https_connections(self) -> Mapping[str, HttpsConnection]:
         return frozendict(
             {
-                key: HttpsConnection(params, self._client_certificate, self._https_proxy_url, self._ca_bundle_path)
+                key: HttpsConnection(
+                    params, self._client_certificate, self._https_proxy_url, self.server_certificates_file_path
+                )
                 for key, params in self._source_parameters.https_connections.items()
             }
         )
 
-    @cached_property
-    def _ca_bundle_path(self) -> Optional[str]:
+    @property
+    def server_certificates_file_path(self) -> Optional[str]:
```

Comment:
last comment sorry, but _technically_ the path can be a directory path instead of a file path (we're just using a file right now, but `verify` takes dir paths as well, so conceivably we could at some point in the future use a folder)
```suggestion
    def server_certificates_bundle_path(self) -> Optional[str]:
```